06-building-a-program-start-to-finish/writing-tests/output/invertedIndexSpec.js

```diff
@@ -0,0 +1,263 @@
+const books = require('../assets/books.json');
+const wrongData = require('../assets/wrongData.json');
+const emptyFile = require('../assets/emptyFile.json');
+const smallFile = require('../assets/smallFile.json');
+
+describe('InvertedIndex Class', () => {
+  beforeAll(() => {
+    this.invertedIndex = new InvertedIndex();
```

Comment:
1. Thanks for pointing that out, i have rectified this issues by declaring it as a `const` variable
2. Thanks for noticing this, yes my inverted index class needs to be instantiated so i can test all the methods in it.